CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+## Contributing
```

Comment:
In primer/brand, I was planning to move this and RELEASING.md into /contributor-docs folder, which is also where all ADRs, RFCs etc live.. shall we do that here too?

Reply:
Sounds good to me. Is there a convention around those files?

Reply:
We can do this later. Not blocking. We don't actually have a contributor-docs section in this repo, so we can do all of that in a separate PR.